conf/nomad/redash_server.hcl

```
@@ -36,6 +36,10 @@ job "redash-server" {
               destination_name = "redash-email-service"
               local_bind_port = 80
             }
+            upstreams {
+              destination_name = "presto"
+              local_bind_port = 8080
```

```suggestion
              destination_name = "${presto_service_name}"
              local_bind_port =  "${presto_port}"
```

Probably will require changes in several places, if in-hurry, just skip it for now

---

Since this was a poc we didnt bother to use variables. If it is desirable that we continue to polish this, we can of course add it.